charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -134,9 +134,9 @@ remote:
 
 scheduler:
   class: "scheduler.periodic.PeriodicScheduler"
-  fit_every: "1d"
-  fit_window: "7d"
-  infer_every: "10s"
+  fit_every: "1h"
+  fit_window: "14d"
+  infer_every: "1m"
```

Comment:
we have 
![image](https://github.com/VictoriaMetrics/helm-charts/assets/11490990/22d8cc5c-3618-4f06-b983-43580e1d5121)
in master, I assume those are even better